src/entities/PoolNetwork.ts

```diff
@@ -82,12 +82,13 @@ export class PoolNetwork extends Entity {
    * Get the deployed Vaults for a given share class. There may exist one Vault for each allowed investment currency.
    * Vaults are used to submit/claim investments and redemptions.
    * @param scId - The share class ID
+   * @param includeDisabled - Whether to include disabled (unlinked) vaults
    */
-  vaults(scId: ShareClassId) {
+  vaults(scId: ShareClassId, includeDisabled = false) {
```

Comment:
It might make more sense to just call this `includeUnlinked`, to not introduce more new terminology that deviates from protocol naming? Unlinked and disabled (linked and enabled) seems both similarly clear/easy to understand to me.